doc-deploy-dev/action.yml

```diff
@@ -306,15 +306,15 @@ runs:
           sed -i 's/src="\([^:"]*\)"/src="version\/stable\/\1"/g' index.html
           sed -i 's/action="search.html"/action="version\/stable\/search.html"/g' index.html
           sed -i 's|script\.src = ".*_static/js/meilisearch_theme_wrap\.js"|script.src = "version/stable/_static/js/meilisearch_theme_wrap.js"|g' index.html
-          sed -i 's|var searchPath = ".*_static/search.json";|var searchPath = "version/stable/_static/search.json";|g' index.html
+          sed -i 's|var SEARCH_FILE = ".*_static/search.json";|var SEARCH_FILE = "version/stable/_static/search.json";|g' index.html
```

Comment:
We are no longer using "var" but "const". We need to update these lines, @Revathyvenugopal162. My fault.